pkg/cluster/cluster_config.go

```diff
@@ -53,16 +53,25 @@ func IsNotReservedNamespace(ns *corev1.Namespace) bool {
 // GetClusterServiceVersion retries the clusterserviceversions available in the operator namespace.
 func GetClusterServiceVersion(ctx context.Context, c client.Client, watchNameSpace string) (*ofapiv1alpha1.ClusterServiceVersion, error) {
 	clusterServiceVersionList := &ofapiv1alpha1.ClusterServiceVersionList{}
-	if err := c.List(ctx, clusterServiceVersionList, client.InNamespace(watchNameSpace)); err != nil {
-		return nil, fmt.Errorf("failed listing cluster service versions: %w", err)
+	listOptions := &client.ListOptions{
```

Comment:
maybe we could give it a bit more descriptive name such as: `paginateNamespacedResult` or something similar.

Reply:
hehe, maybe mixed function, but here is for CSV not for the Namespace :coffee:

Reply:
Correct, but it still holds:)